.gitignore

```diff
@@ -174,3 +174,5 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+/.vs
```

Comment:
This is causing a merge conflict for some reason, I'd remove it and make a separate quick PR to fix this (can you can merge develop into feature to get it)

Reply:
oh that pr already exists, already merged into develop

Reply:
Is there a reason why the `.gitignore` is here then? If not, remove it from the PR?

Reply:
made a pr, merged into feature/behavior, closed pr